File: DE3.WebServices.Back/DE3.WebServices.Back.TimeService/src/main/java/de3/webservices/back/timeservice/model/TimeSlot.java
package de3.webservices.back.timeservice.model;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;
import javax.persistence.ManyToOne;

@Entity

public class TimeSlot {
	
	public @Id @GeneratedValue Long IdSlot;
	
	public Long HourCount;
	public Long IdUser;
	public Long IdProject;
	@ManyToOne
	public Dates ReferredDate;
}

File: DE3.WebServices.Back/DE3.WebServices.TimeService.Api/WebApplication1/DataAccess/Model/Timeslot.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace WebApplication1.DataAccess.Model
{
    public partial class Timeslot
    {
        public int IdSlot { get; set; }
        public int HourCount { get; set; }
        public int IdUser { get; set; }
        public int IdProject { get; set; }
        public DateTime ReferredDate { get; set; }
    }
}
